Week4/managementsystem.ipynb

```diff
@@ -0,0 +1,371 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "019de1d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as sq\n",
+    "import datetime"
```

Comment:
Without importing datetime module, if i want to display datetime in code, how  can i do?

Reply:
import time

current_time = time.localtime()
formatted_time = time.strftime("%Y-%m-%d %H:%M:%S", current_time)
print("Current date and time:", formatted_time)